src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue

```diff
@@ -219,8 +219,8 @@ export default {
 			}
 
 			if (this.currentParticipantIsModerator) {
-				if (participant1.attendeePermissions !== participant2.attendeePermissions) {
-					return participant1.attendeePermissions < participant2.attendeePermissions ? 1 : -1
+				if (participant1.permissions !== participant2.permissions) {
```

Comment:
From my pov this is intentional so custom permissions are sorted over the users that fall back to default permissions